subprojects/docs/src/snippets/scala/zincDependency/groovy/build.gradle

```diff
@@ -7,11 +7,11 @@ repositories {
 }
 
 dependencies {
-    implementation 'org.scala-lang:scala-library:2.12.1'
+    implementation 'org.scala-lang:scala-library:2.13.8'
```

Comment:
Since 2.12.1 is ancient at this point, I thought it would make sense to update to something more recent. Please let me know if I should revert this and the similar change in the kotlin file.